proto/checkers/stored_game.proto

```diff
@@ -15,4 +15,5 @@ message StoredGame {
   string moveCount = 7;
   string beforeId = 8; // Pertains to the FIFO. Towards head.
   string afterId = 9; // Pertains to the FIFO. Towards tail.
+  string deadline = 10;
```

Comment:
could use https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#duration if its easier

Reply:
Mmh, deadline is an absolute point in time, but duration is a relative length. So not sure how I could shoehorn it in.

Reply:
Mmh, deadline is an absolute point in time, but duration is a relative length. So not sure how I could shoehorn that in.

Reply:
there is also timestamp https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#timestamp

Reply:
Sounds interesting. I tried the following in `proto/checkers/stored_game.proto:

```protobuf
import "google/protobuf/timestamp.proto";
...
google.protobuf.Timestamp deadline = 10;
```
and, after fixing the Go code where necessary, in the auto-generated `x/checkers/types/stored_game.pb.go`:

```go
size, err := m.Deadline.MarshalToSizedBuffer(dAtA[:i])
```
 I got errors:

```
m.Deadline.MarshalToSizedBuffer undefined (type *timestamppb.Timestamp has no field or method MarshalToSizedBuffer)
```
Could this be because Starport does not recognise this type?